src/layouts/sidebar/DropDown.tsx

```diff
@@ -0,0 +1,50 @@
+import { Avatar, IconButton, Menu, MenuItem } from "@mui/material";
+import { UserContext } from "App";
+import { Button } from "modules/shared/Button";
+import { useContext, useState } from "react";
```

Comment:
can we follow specific order for imports.
1) Third party imports
2) Local modules/components
3) Helper files/ css files